app/lib/synthetic_branding_group_populator.rb

```diff
@@ -55,12 +59,14 @@ def self.create_branding_group(branding_group_config, user)
     end
 
     branding_group = BrandingGroup.new(branding_group_config.dig('branding_group'))
-    branding_group.user ||= user
+    branding_group.user_ids = [user.id]
```

Comment:
I think it's generally considered better form to operate on the collections where possible, as that makes the corresponding triggers more efficent.
```suggestion
    branding_group.users = [user]
```